docs/examples/101/api-management-modular/groups.bicep

```diff
@@ -0,0 +1,32 @@
+// ****************************************
+// Azure Bicep Module:
+// Create multiple groups from sample JSON records
+// ****************************************
+@minLength(1)
+param apimInstanceName string
+var groupsSet = [
+  {
+    'groupName': 'APIMGroup1'

```

Comment:
is the reason for quotes on the property names (i.e. `'groupName'`) because this was converted using a tool? Quotes on property names is not required.

Reply:
Done and rectified/ Still need quotes on the values of JSON so as not to confuse with variable names.